### DISCUSSION TITLE
CSS Color attribute not working

### DISCUSSION THREAD

**Discussion by JEEVAN28** [apache/superset#27789] [General]
### Bug description

[data-test-chart-id="96"] .chart-container{
    background-color: green;
    color : white; 
    text-align: center;
}

in above css code, background color is changed but text color and text align is not changing.

### How to reproduce the bug

go to edit css for dashboard

### Screenshots/recordings

![image](https://github.com/apache/superset/assets/12723595/a9e01fca-93e9-421b-9f28-66453f830ae5)

### Superset version

3.1.1

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rusackas** [comment]
I believe the text is in a separate nested div, and you'll just need another selector... please check that in the DOM to confirm.
